config/pipelines/bespoke_chromium_5p.yml

```diff
@@ -15,4 +15,5 @@ Bespoke Chromium 5p MX:
       - limber_multiplexing
   relationships:
     LBC 5p GEX PCR 2XP: LBC 5p GLibPS
-    LBC 5p GLibPS: LB Lib Pool Norm
+    LBC 5p GEX PCR 2XP: LBC 5p Pool Norm
```

Comment:
Should be LBC 5p GLibPS: LBC 5p Pool Norm

Reply:
Yep thanks!